local_settings.sh.in

```diff
@@ -303,8 +303,34 @@ uninstall_policies() {
 	_restore_file_modes
 }
 
+verify_policies() {
+	if [[ ! selinuxenabled ]]; then
+		echo "SELinux is disabled"
+		exit 0
+	fi
+	if [ $(getenforce) != Enforcing ]; then
+		echo "SELinux is permissive";
+		exit 0
+	fi
+
+	failed_count=0
+	for module in $MODULES; do
+		local_failed=1
+		while read installed_module; do
```

Comment:
If you wanted to, you could reduce fork/execs by placing "INSTALLED_MODULES=$(semodule -l)" above line 317 and changing your inner loop to:

for installed_module in $INSTALLED_MODULES; do
  ...
done

I did mention the < <() thing on IRC, but I think I assumed the loops would be the other way (which would complicate the logic a lot, and is a bad idea).

Reply:
right - updated!